content/docs/reference/microcontrollers/nano-rp2040.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Nano RP2040"
+weight: 3
+---
+
+The [Nano RP2040]() is a tiny development board based on the Raspberry Pi [RP2040]() microcontroller.
+
+## Interfaces
+
+| Interface | Hardware Supported | TinyGo Support |
+| --------- | ------------- | ----- |
+| GPIO      | YES | YES |
+| UART      | YES | YES |
+| SPI      | YES | YES |
+| I2C      | YES | YES |
+| ADC      | YES | YES |
+| PWM      | YES | YES |
+
+## Machine Package Docs
+
+[Documentation for the machine package for the Nano RP2040](../machine/nano-rp2040)
+
+## Flashing
+
+### UF2
+
+The Nano RP2040 comes with the [UF2 bootloader](https://github.com/Microsoft/uf2) already installed.
+
+### CLI Flashing
+
+- Plug your Nano RP2040 into your computer's USB port while holding down the RESET button on the board.
```

Comment:
Not exactly. Need to short REC and GND pins. A hassle :(

Reply:
It was perhaps premature to add this board when the vast majority of users will not be able to flash it. This may result in a lot of extra support needed.